storage/integration_test.go

```diff
@@ -5307,6 +5312,27 @@ func (h testHelper) mustRead(obj *ObjectHandle) []byte {
 	return data
 }
 
+// deleteObjectIfExists deletes an object with a RetryAlways policy (unless another
```

Comment:
It seems like we should maybe be doing this for object cleanup in general? Wondering if there are other tests which should also use this method.

Reply:
I believe we have a similar method that does a lot of our object cleanups, perhaps I can merge the two.

Reply:
Feel free, but it can be in a separate PR if that's easier

Reply:
Sounds good, I will merge this as-is to get the test fix in.